controllers/pod_models.go

```diff
@@ -1250,6 +1256,39 @@ func NormalizeClusterSpec(spec *fdbtypes.FoundationDBClusterSpec, options Deprec
 	return nil
 }
 
+// GetPodDisruptionBudgets generates the disruption budgets that we need to
+// configure for a cluster.
+func GetPodDisruptionBudgets(cluster *fdbtypes.FoundationDBCluster) ([]policyv1beta1.PodDisruptionBudget, error) {
```

Comment:
I would move this code part into `update_pdbs.go` I don't think that we should keep it in `pod_models`?